### ISSUE TITLE
Invalid MalUrl models for relations

### ISSUE THREAD

**Issue by ToshY** [jikan-me/jikan#402]
**Problem**

Show relations still contain bugged links, returning `MalUrl` models when they are invalid.

**Reproduction**

For show [918](https://api.jikan.moe/v3/anime/918), under `related`, `Alternative version`.

Json response from API

```json
{
   "Alternative version":[
      {
         "mal_id":7472,
         "type":"anime",
         "name":"Gintama Movie 1: Shinyaku Benizakura-hen",
         "url":"https:\/\/myanimelist.net\/anime\/7472\/Gintama_Movie_1__Shinyaku_Benizakura-hen"
      },
      {
         "mal_id":21899,
         "type":"anime",
         "name":"Gintama: Yorinuki Gintama-san on Theater 2D",
         "url":"https:\/\/myanimelist.net\/anime\/21899\/Gintama__Yorinuki_Gintama-san_on_Theater_2D"
      },
      {
         "mal_id":0,
         "type":"anime",
         "name":"",
         "url":"https:\/\/myanimelist.net\/anime\/\/"
      }
   ]
}
```

Debug in application

```php
-related: array:6
    "Adaptation" => array:1
    "Side story" => array:5
    "Alternative version" => array:3
      0 => [Jikan\Model\Common\MalUrl]()
      1 => [Jikan\Model\Common\MalUrl]()
      2 => [Jikan\Model\Common\MalUrl]()
        -name: ""
        -url: "https://myanimelist.net/anime//"
      }
    ]
    "Sequel" => array:1
    "Other" => array:3
    "Character" => array:1
  ]
```

**Expectation**

No `MalUrl` model for invalid entries.

**Extra info**

Seems related to #200 and the [following changes](https://github.com/jikan-me/jikan/commit/140b656ebfab49ad21a470367769dd31d5c98c78).

**Version**
`v3.0.0-beta.9`

**Comment by irfan-dahir** [comment]
Hi sorry for the late response. 

`v3.0.0-beta.9`
You're on a really old version. Update to [v3.0.0-rc.3](https://github.com/jikan-me/jikan/releases/tag/v3.0.0-rc.3). This issue doesn't exist there.

Alternatively, you can update to [v3.0.0-rc.5](https://github.com/jikan-me/jikan/releases/tag/v3.0.0-rc.5) but it's php^8 only.

**Comment by ToshY** [comment]
Hey @irfan-dahir

No problem! I couldn't upgrade and test it in the latest release candidate as it had dependency conflict as stated in #395. I see in the comments that it has been resolved in [v3.0.0-rc.5](https://github.com/jikan-me/jikan/releases/tag/v3.0.0-rc.5), so I can use that version indeed. 

Thanks!
